### ISSUE TITLE
Are you still doing this project?

### ISSUE THREAD

**Issue by xfce** [pa3gsb/RadioBerry#1]
Hi 
Are you still doing this project, I am also interested in HAM, and if you want, maybe I could do some hardware job, like PCB design.

**Comment by jellemaas** [comment]
Yes iam working at a version 2 which is in beta 2 now. Designed by using KICAD.

I will provide all the info here at github!

**Comment by xfce** [comment]
@jellemaas  OK . but I am only familiar with Altium Designer, know little of KICAD^-^

**Comment by spikebike** [comment]
Any update on the progress?  Is there a URL with related discussion and specifications?  Looks like a very interesting project.

**Comment by pa3gsb** [comment]
all information is at github. or you can look at www.pa3gsb.nl
